test/integration/targets/tower_credential_type/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
In keeping with similar integration tests, could we also add a test that removes this credential type via `state: absent` ?

Thanks!

Reply:
No worries, I've added a test with a `state: absent` to remove the credential type.